src/Microsoft.OData.ModelBuilder/Vocabularies/VocabularyTermConfiguration.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Vocabularies;
+
+namespace Microsoft.OData.ModelBuilder
+{
+    /// <summary>
+    /// Base vocabulary configuration.
+    /// </summary>
+    public abstract partial class VocabularyTermConfiguration : IRecord
+    {
+        private readonly string _termName;
```

Comment:
>private readonly string _termName; [](start = 8, length = 34)

It seems we can mark the "_termName" as abstract property in this class

public abstract string TermName {get;}


in each derived class, we can override it and directly return the term name string.

for example in AcceptableEncodingsConfiguration.cs

public override string TermName => "Org.OData.Capabilities.V1.AcceptableEncodings";

Reply:
👍